### PR TITLE
Documenting how fixtures can be moved from /src/DataFixures to a custom directory

### PR DESCRIPTION
**Pull request by sebastianstucke87**
Doctrine fixtures can be loaded from any directory/file (https://www.doctrine-project.org/projects/doctrine-data-fixtures/en/latest/how-to/loading-fixtures.html#loading-fixtures). 

But it is not clear on how to do this with the DoctrineBundle:

1. "How to Override Symfony's default Directory Structure"
    - https://symfony.com/doc/current/configuration/override_dir_structure.html
    - No word of fixtures

2. "How to Override any Part of a Bundle"
    - https://symfony.com/doc/current/bundles/override.html
    - No mentions of directories/fixtures

3. "DoctrineFixturesBundle" (this repository)
    - https://symfony.com/bundles/DoctrineFixturesBundle/current/index.html
    - No option/configuration mentioned to customize the "DataFixtures"-directory.

### CODE REVIEW

**Comment by derrabus** [comment]
Thank you for your PR. Please try to find a PR title that describes and summarizes what the PR is doing.

File: Resources/doc/index.rst [hunk 1]
```diff
@@ -384,3 +384,43 @@ With the ``--purger`` option we can now specify to use ``my_purger`` instead of
...
+    namespace Symfony\Component\DependencyInjection\Loader\Configurator;
+
+    return function(ContainerConfigurator $configurator) : void {
+        $services = $container->services()
+            ->defaults()
+                ->autowire()
+                ->autoconfigure();
+
+        $services->load('DataFixtures\\', '../fixtures');
+    };
```
Comment (SenseException) [inline Resources/doc/index.rst]: Is `$configurator` and `$container` supposed to be the same variable?

Comment (sebastianstucke87) [reply Resources/doc/index.rst]: Good catch! I fixed it.

File: Resources/doc/index.rst [hunk 2]
```diff
@@ -384,3 +384,43 @@ With the ``--purger`` option we can now specify to use ``my_purger`` instead of
 
 .. _`Doctrine ORM`: https://symfony.com/doc/current/doctrine.html
 .. _`DoctrineMongoDBBundle`: https://github.com/doctrine/DoctrineMongoDBBundle
+
+
+How to load Fixtures from a different Directoy
+---------------------------
```
Comment (SenseException) [inline Resources/doc/index.rst]:
```suggestion
How to load Fixtures from a different Directory
-----------------------------------------------
```

**Review by SenseException — CHANGES_REQUESTED** [review]
@derrabus What do you think about the content of the additional docs for a new file path?

File: Resources/doc/index.rst [hunk 3]
```diff
@@ -384,3 +384,43 @@ With the ``--purger`` option we can now specify to use ``my_purger`` instead of
...
+By default, fixtures are loaded from the ``/src/DataFixtures``-directory.
```
Comment (greg0ire) [inline Resources/doc/index.rst]:
```suggestion
By default, fixtures are loaded from the ``src/DataFixtures`` directory.
```

File: Resources/doc/index.rst [hunk 4]
```diff
@@ -384,3 +384,43 @@ With the ``--purger`` option we can now specify to use ``my_purger`` instead of
...
+In this example, we are going to load our DataFixtures from a new ``/fixtures`` directory.
```
Comment (greg0ire) [inline Resources/doc/index.rst]:
```suggestion
In this example, we are going to load our DataFixtures from a new ``fixtures`` directory.
```

File: Resources/doc/index.rst [hunk 5]
```diff
@@ -384,3 +384,43 @@ With the ``--purger`` option we can now specify to use ``my_purger`` instead of
...
+
+First, add a new ``PSR-4`` autoload-entry in the ``composer.json`` with the new ``/fixtures`` directory:
```
Comment (greg0ire) [inline Resources/doc/index.rst]:
```suggestion
First, add a new ``PSR-4`` autoload-entry in the ``composer.json`` with the new ``fixtures`` directory:
```

File: Resources/doc/index.rst [hunk 6]
```diff
@@ -384,3 +384,43 @@ With the ``--purger`` option we can now specify to use ``my_purger`` instead of
...
+.. code-block:: json
+
+    "autoload-dev": {
+        "psr-4": {
+            "...": "...",
+            "DataFixtures\\": "fixtures"
+        }
+    },
+
+Then, enable Dependency Injection for the ``/fixtures`` directory:
```
Comment (greg0ire) [inline Resources/doc/index.rst]:
```suggestion
Then, enable Dependency Injection for the ``fixtures`` directory:
```

File: Resources/doc/index.rst [hunk 7]
```diff
@@ -384,3 +384,43 @@ With the ``--purger`` option we can now specify to use ``my_purger`` instead of
...
+            ->defaults()
+                ->autowire()
+                ->autoconfigure();
+
+        $services->load('DataFixtures\\', '../fixtures');
+    };
+
+.. caution::
+
+    This will not override the default ``/src/DataFixtures`` directory when creating fixtures with the
```
Comment (greg0ire) [inline Resources/doc/index.rst]:
```suggestion
    This will not override the default ``src/DataFixtures`` directory when creating fixtures with the
```

**Review by SenseException — APPROVED** [review]

**Comment by greg0ire** [comment]
Thanks @sebastianstucke87 !

**Pull request merged**